blocks/quote/quote.js

```diff
@@ -1,20 +1,30 @@
 export default async function decorate(block) {
-  const [quotation, attribution] = [...block.children].map((c) => c.firstElementChild);
+  const [quotation, attribution, attributionUrl] = [...block.children].map((child) => child.firstElementChild);
```

Comment:
Could we go another step further and remove the nested `div`s such that the final output of this block is:

```html
<blockquote>
  <p class="quote__quotation">...</p>
  <p class="quote__attribution">
    <cite>...</cite>
  </p>
</blockquote>
```

I moved around the structure a bit there - we can nest a `<cite>` within a `<p>` tag, but not the other way around. Let's also update the CSS class names to BEM-style naming.

Reply:
Great call